readme.md

```diff
@@ -37,7 +38,17 @@ public function register_post_types() {
   );
 }
 
+/**
+ * Register taxonomies
+ *
+ * @return void
+ */
+public function register_taxonomies() {
```

Comment:
Same as for the CPT example, I think we should keep the example simple by dropping the class presentation:

```php
function THEME_register_taxonomies() {
    // ...
}

add_action( 'init', 'THEME_register_taxonomies' );
```